pkg/docker/docker_api.go

```diff
@@ -0,0 +1,283 @@
+package docker_api
```

Comment:
package name should match with directory structure. use `package docker`.
- if you want package name as `dockerapi` not `docker_api`